### DISCUSSION TITLE
belchertown-dark.min.css

### DISCUSSION THREAD

**Discussion by roe-dl** [poblabs/weewx-belchertown#887] [Q&A] [answered]
Is there a possibility to get the source of `belchertown-dark.min.css`? 

The reason why I ask is, that within that style sheet there are some clauses that destroy the Bootstrap grid system. So I would like to edit the file.

**Comment by poblabs** [comment, marked as answer]
I'm pretty certain the base of the css file is this theme from bootswatch: https://bootswatch.com/darkly/

Then after that is Highcharts Dark Unica theme. I can't find the source but I did find this codepen. The source might be built into the older version of highcharts/highstock? I can't recall. https://codepen.io/larsac07/pen/GqWaBg

Then after that is the belchertown skin overrides. 

![image](https://user-images.githubusercontent.com/3484775/236552727-20c9fc29-c33a-4740-9d2d-39228d707e71.png)

**Reply by poblabs** [reply]
You'll have to turn on word wrapped because it's a minified version of the css, so each line is very long
